packages/p2p-v2/src/components/Wizard/WizardStep.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { PropsWithChildren } from 'react';
+import clsx from 'clsx';
+
+const Step = ({ children = [], className }: PropsWithChildren<{ className?: string }>) => (
+    <div className={clsx('.p2p-v2-wizard__main-step', className)}>{children}</div>
```

Comment:
where is the class `p2p-v2-wizard__main-step` instantiated? cause i don't see it inside of the Wizard.scss file. I also saw below you target the class to change the padding on mobile... can we create a scss file for this component or inside of Wizard.scss and add the mobile styles there instead of directly targeting the class

Reply:
yes. will do. btw The Wizard, Wizard step and FormProgress will be replaced once the formprogress from deriv-com/ui(from tradershub) will be merged, for now, this is just a copy from existing shared components and i haven't done much of a refactoring on it since it will be deleted soon.

Reply:
ah got it 👍